### PR TITLE
removed NANs and isnans from CRVResponse modules

### PR DESCRIPTION
**Pull request by ehrlich-uva**

### CODE REVIEW

**Comment by ehrlich-uva** [comment]
I would like to have this PR merged separately so that we would notice any potential differences in the nightly validations. (It shouldn't result in any differences.)

**Comment by kutschke** [comment]
> I would like to have this PR merged separately so that we would notice any potential differences in the nightly validations. (It shouldn't result in any differences.)

Will do.

File: CRVResponse/src/MakeCrvRecoPulses.cc [hunk 1]
```diff
@@ -277,7 +278,8 @@ void MakeCrvRecoPulses::SetWaveform(const std::vector<unsigned int> &waveform,
     double pulseTime    = fitParam1;
     float  pulseHeight  = fitParam0/TMath::E();
     float  pulseBeta    = fitParam2;
-    float  pulseFitChi2 = (fr->Ndf()>0?fr->Chi2()/fr->Ndf():NAN);
+    float  pulseFitChi2 = (fr->Ndf()>0?fr->Chi2()/fr->Ndf():-1);
+    bool   zeroNdf      = (fr->Ndf()>0?0:1);
```
Comment (rlcee) [inline CRVResponse/src/MakeCrvRecoPulses.cc:282]: I'd use false:true here for readability

**Review by rlcee — APPROVED** [review]
Approved, with one note for your consideration

File: CRVResponse/src/MakeCrvRecoPulses.cc (hunk 1 above)
Comment (ehrlich-uva) [reply CRVResponse/src/MakeCrvRecoPulses.cc:282]: I will change it.

**Comment by kutschke** [comment]
@FNALbuild run build test

**Pull request merged**